src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/Tomcat7Container/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # The admin user has username 'admin' and password 'tomcat'
 #
 
-FROM ubuntu:22.04
+FROM ubuntu:xenial
```

Comment:
That change worries me: https://releases.ubuntu.com/16.04/ is not maintained since 2019 (and 18.04 is almost end of life).

Can we (jenkins-infra) help on the packages versions part?

Reply:
@dduportal I would be forever in your debt if you can take a look at the ssh issues on `GitPluginTest` when using 22.04 as base. I believe I can workaround the issue on the `Tomcat7Container` myself

Reply:
Gotta try, but I need to (re)learn how it works (expect some overhead ;) )